cloud/aws/main.tf

```diff
@@ -83,7 +84,7 @@ resource "null_resource" "cockroach-initializer" {
     inline = [
       "bash download_binary.sh cockroach/cockroach ${var.cockroach_sha}",
       "mkdir -p logs",
-      "./cockroach init --logtostderr=true --stores=sdd=data",
+      "./cockroach init --logtostderr=true --stores=${element(split(",", "${var.stores}"), 0)}",
```

Comment:
as for the `init` step, it can use `element(0,...)`, and the `start` step can keep all of them.